app/assets/stylesheets/components/_border.scss

```diff
@@ -3,6 +3,8 @@
 .bw3 { border-width: 3px; }
 .bw4 { border-width: 4px; }
 
+.br-md { border-radius: $border-radius-md; }
```

Comment:
Looks like we have two naming conventions `br-md` and `rounded-lg`.  `rounded` is the default basscss name so that's why I went with `rounded-lg` but really don't mind as long as we keep it consistent.